repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -54,7 +54,7 @@ class ExecutorClassLoader(
     classUri: String,
     parent: ClassLoader,
     userClassPathFirst: Boolean) extends ClassLoader(null) with Logging {
-  val uri = new URI(classUri)
```

Comment:
@guolianwei plase

1. file a JIRA
2. describe how you reproduced the issue and tested this patch
3. add a unit test.

See also https://spark.apache.org/contributing.html